tests/integration/test_dynamodb.py

```diff
@@ -1016,16 +999,12 @@ def test_batch_write_items(self):
         )
         assert response["ResponseMetadata"]["HTTPStatusCode"] == 200
 
-        # clean up
-        dynamodb.delete_table(TableName=table_name)
-
-    def test_dynamodb_stream_records_with_update_item(self, dynamodb):
+    def test_dynamodb_stream_records_with_update_item(self, dynamodb_client, dynamodb_create_table):
```

Comment:
```suggestion
    def test_dynamodb_stream_records_with_update_item(self, dynamodb_client, dynamodb_create_table, dynamodb):
```
`dynamodb` fixture is still used in line 1011. This should also fix the build :)

Reply:
Thanks! I guess that will teach me to put my hands in tests, there are still some more to refactor to use fixtures